IFC4x3/Properties/f/FaultInfilling_2JZfahd4z6yxCGa85SabRQ/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FaultInfilling_2JZfahd4z6yxCGa85SabRQ" Name="FaultInfilling" UniqueId="938e992b-9c4f-46f3-b310-90815c9256da" PrimaryDataType="IfcIdentifier" />
```

Comment:
I would propose `IfcLabel` as a datatype. Typically `IfcIdentifier` is a machine readable identifier, whereas `IfcLabel` is human readable.